elytron/WFLY-16532-additional-scope-for-auth-request.adoc

```diff
@@ -0,0 +1,129 @@
+== Adding the ability to configure additional scope for authentication request
+:author:            Prarthona Paul
+:email:             prpaul@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+OpenID Connect is an authentication mechanism that builds on OAuth 2.0 
+and allows a user to login to a web application using credentials established 
+by an OpenID provider. 
+Currently, when sending an authentication request to the OpenID provider, one 
+of the required parameters with the authentication flow is "scope". However, for 
+now, that value is hardcoded as just "openid". 
+
+The specifications indicate that there are other scope values which may be included in 
```

Comment:
s/specification/OpenID Connect specification

Would be good to add the link to the relevant section as well:
https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest

Reply:
Would also be good to check if the specification mentions anything about the values that are allowed for the scope values.